api/discounts-get/index.js

```diff
@@ -1,5 +1,10 @@
 const data = require('../shared/discount-data');
 
+/**
+ * Retrieves discounts and sends them as a JSON response.

```

Comment:
Apply these docs to the other APIs too. For example: https://github.com/ntrogh/shopathome/blob/7e5a15d8a563c0df352aab28ac27628401f4859e/api/products-delete/index.js#L3-L4